### DISCUSSION TITLE
illumna short reads

### DISCUSSION THREAD

**Discussion by tky199996** [ConesaLab/SQANTI3#289] [General] [answered]
Hello, I have been using SQANTI3 for analysis recently, I have full-length transcriptome data for 7 tissues, but my short-read sequencing data is only for 5 tissues, is this in line with the requirements of SQANTI3 run?

**Comment by carolinamonzo** [comment, marked as answer]
Hi @tky199996 that's up to you and your biological question. You can use sample-paired short and long-reads data, or public short reads with your long-reads data. This is not a SQANTI issue, it depends on your research aims.

**Comment by Sparkle-27** [comment]
> Hi @tky199996 that's up to you and your biological question. You can use sample-paired short and long-reads data, or public short reads with your long-reads data. This is not a SQANTI issue, it depends on your research aims.

Hi @carolinamonzo , I have sequenced 3 samples with long reads and short reads, respectively. In addition, more than 20 samples with the same tissue were under Illumina short reads. Would the gff results more accurate by performing SQANTI3 QC --short_reads with short reads as much as possible . Or just 3 long reads and corresponding short reads would be OK for the pipeline ?
Best

**Comment by carolinamonzo** [comment]
Hi @Sparkle-27, it depends on your biological question. If you are only trying to work on your specific sample's information, you should run SQANTI3 QC with the short reads of that sample. However, if you want to check wether there is a possibility that some splice junctions seen in your long reads data could actually exist even though they are not present in your short reads data (maybe because of low depth of sequencing or little diversity in your starting RNA?), then I would use all the other samples as well to complement the long reads.
As I said before to Tky, it all depends on the question you want to answer.

**Reply by Sparkle-27** [reply]
THX @carolinamonzo , I forgot to reply. I have another question, the tpm isoform expression matrix inferred from SQANTI3 built-in salmon, while I still found a lot of isoforms without expressing (TPM=0),  which was defined as noise isoform in https://doi.org/10.1038/s41587-024-02245-9.  Why SQANTI3 not  filter these isoforms?

**Reply by carolinamonzo** [reply]
Hi @Sparkle-27, SQANTI3_QC module is only doing the quality control, combination of the orthogonal data and checking the support. To filter those isoforms, there's the module SQANTI3_filter. You can check out how it works and how to run it here: https://github.com/ConesaLab/SQANTI3/wiki/Running-SQANTI3-filter
